TrafficCapture/replayerPlugins/jsonMessageTransformers/Es2OsPost23/src/main/java/org/opensearch/migrations/transform/Es2OsPost23.java

```diff
@@ -1,9 +1,30 @@
 package org.opensearch.migrations.transform;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class Es2OsPost23 implements IJsonTransformerProvider {
```

Comment:
Please affix this with TransformerProvider and override getName() to get the value that you want.